.github/workflows/build.yml

```diff
@@ -99,7 +99,7 @@ jobs:
 
   sanitize:
     name: sanitize
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04
```

Comment:
Why this?

Reply:
pulls in newer clang. (17)

Reply:
see also the relevant CI run: https://github.com/WebAssembly/wabt/actions/runs/10896075346/job/30235292696

Reply:
I see.  SGTM..  I wonder wonder why `ubuntu-latest` is not already 24.04?

Reply:
:shrug: github takes a while to roll out these things. we should set it back when `ubuntu-latest` becomes `24.04`.